tests/test_autonomy/test_cli/test_helpers/test_docstrings.py

```diff
@@ -0,0 +1,99 @@
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
+#
+#   Copyright 2022 Valory AG
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+# ------------------------------------------------------------------------------
+
+"""The module contains test_helpers for module tests."""
+
+import shutil
+import tempfile
+from pathlib import Path
+from types import ModuleType
+from typing import cast
+from unittest import mock
+
+import pytest
+from _pytest.capture import CaptureFixture  # type: ignore
+
+from autonomy.cli.helpers.docstring import analyse_docstrings, import_rounds_module
+
+import packages
+from packages.valory.skills import test_abci
+from packages.valory.skills.test_abci import rounds as test_abci_rounds
+
+
+@pytest.mark.parametrize("module", [test_abci, test_abci_rounds])
+def test_import_rounds_module(module: ModuleType) -> None:
+    """Test import_rounds_module"""
+
+    module_path = Path(module.__file__)
+    module = import_rounds_module(module_path)
+    assert module is test_abci_rounds
+
+
+def test_import_rounds_module_failure() -> None:
+    """Test import_rounds_module"""
+
+    with pytest.raises(ModuleNotFoundError, match="No module named 'packages.rounds'"):
+        module_path = Path(packages.__file__)
+        import_rounds_module(module_path)
+
+
+def test_import_rounds_module_with_non_default_package_dir() -> None:
+    """Test import_rounds_module from different package directory"""
+
+    with tempfile.TemporaryDirectory() as tmp_dir:
+        tmp_path = Path(tmp_dir)
+        module_path = Path(packages.__file__)
+        shutil.copytree(module_path.parent, tmp_path / packages.__name__)
+        new_package_dir = tmp_path / module_path.parts[-1]
+        module = import_rounds_module(module_path, packages_dir=new_package_dir)
+        assert module.__file__.startswith(tmp_dir)
```

Comment:
This will fail for the following reasons:
1. resolving the path won't work
2. importlib.importmodule doesn't work like this

https://github.com/valory-xyz/open-autonomy/blob/c3c82f868bcff341e31d4ea222a1448fdcc6eaf0/autonomy/cli/helpers/docstring.py#L37-L45

Reply:
1. Yeah it won't work because `module_path` is supposed to be a subdirectory of the `packages_dir`

If you look at the command definition, we clearly derive the module path from the packages dir

https://github.com/valory-xyz/open-autonomy/blob/80fbbe571d49b802e945b9ade64d69a390505151/autonomy/cli/analyse.py#L135

And even if the resolving worked somehow, the test will still fail because neither packages dir or the temp dir will directly contain any `rounds.py` file

The purpose of this function is to import the `rounds.py` module for given skill package, which is by default will be a subdirectory of the given packages directory. 

2. How does it work?